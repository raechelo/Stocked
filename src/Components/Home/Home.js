import React, { Component } from 'react';
import Form from '../../Containers/Form/Form';
import Footer from '../../Components/Footer/Footer';
import Header from '../../Components/Header/Header';
import Instructions from '../../Components/Instructions/Instructions';

class Home extends Component {
  constructor() {
    super();
    this.state = {
      isExpanded: false
    }
  }

  render() {
    return (
      <section>
          <Header />
          <Form />
          <Instructions />
          <Footer />
      </section>
    )
  }
}

export default Home;