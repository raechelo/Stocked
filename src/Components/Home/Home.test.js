import { Home } from './Home';
import { shallow } from 'enzyme';
import React from 'react';

describe('Home', () => {

  let wrapper;
  let recipes = [{title: 'Mac&Cheese'}]

  beforeEach(() => {
    wrapper = shallow(<Home recipes={recipes} />)
  });

  it('should match the snapshot with all the data passed in', () => {
    expect(wrapper).toMatchSnapshot();

    wrapper.setState({recipes: []})
    expect(wrapper).toMatchSnapshot();
  });

});