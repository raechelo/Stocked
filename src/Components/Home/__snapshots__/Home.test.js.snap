// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match the snapshot with all the data passed in 1`] = `ShallowWrapper {}`;

exports[`Home should match the snapshot with all the data passed in 2`] = `ShallowWrapper {}`;
